tcms/testruns/migrations/0012_test_execusion_add_fields.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.1.6 on 2021-02-17 04:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("testruns", "0011_add_planning_fields"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="historicaltestexecution",
+            old_name="close_date",
+            new_name="stop_date",
+        ),
+        migrations.RenameField(
+            model_name="testexecution",
+            old_name="close_date",
+            new_name="stop_date",
+        ),
```

Comment:
The renamed field also has `db_index=True`, see models.py which means the migration also needs an AlterField operation on these fields to add the index.

Reply:
When I added the additional migrations:
`
        migrations.AlterField(
            model_name="testexecution",
            name="stop_date",
            field=models.AutoField(primary_key=True, serialize=False),
        ),
        migrations.AlterField(
            model_name="testexecution",
            name="start_date",
            field=models.AutoField(primary_key=True, serialize=False),
        ),
`
there pops up an error that "AssertionError: Model testruns.TestExecution can't have more than one auto-generated field." do you have any advice as to where I should look to make it work?

Reply:
The error is self explanatory, you can't have 2 primary keys in a table. 

Before going further you should think about what you are trying to do and not try to guess it because I see that it isn't very clear to you what you are doing. 

1) My last comment talks about only the renamed field, which is now called `stop_date`. Your last example tries to modify 2 fields instead of one !!! Pay attention to this!

2) The immediate goal for `stop_date` field after it was renamed is to instruct the database engine to create an index for that field. Index has been missing up until now. IDK how you came up with the idea that modifying the field to be a primary key will alter its index but it's not what you want to do. Explore the other migrations in the source code, there are   at least 2 examples of the same operation.  Also read the documentation about AlterField -> https://docs.djangoproject.com/en/3.1/ref/migration-operations/#alterfield